dist/src/main/java/io/camunda/application/StandaloneSchemaManager.java

```diff
@@ -104,7 +113,7 @@ public void run(final String... args) throws Exception {
   // TODO: Use unified configuration when it is available
   @EnableConfigurationProperties(LegacyBrokerBasedProperties.class)
   @ComponentScan(
-      basePackages = "io.camunda.application.commons.search",
+      basePackages = {"io.camunda.application.commons.search", "io.camunda.configuration"},
```

Comment:
:question: Since it is now getting the unified configuration, should the above import to legacy broker based properties be removed?

Reply:
Yeah I think so